services/director-v2/src/simcore_service_director_v2/modules/osparc_variables/_api_auth_rpc.py

```diff
@@ -26,10 +26,11 @@ async def get_or_create_api_key_and_secret(
     rpc_client = get_rabbitmq_rpc_client(app)
     result = await rpc_client.request(
         WEBSERVER_RPC_NAMESPACE,
-        TypeAdapter(RPCMethodName).validate_python("get_or_create_api_key"),
+        TypeAdapter(RPCMethodName).validate_python("create_api_key"),
         user_id=user_id,
         display_name=display_name,
         expiration=expiration,
         product_name=product_name,
+        raise_on_conflict=False,
     )
     return ApiKeyGet.model_validate(result)
```

Comment:
but here you will not get the right secret, right?

Reply:
If you resolve the comment and there is a question, can you at least please answer the questions :-) thx

Reply:
ok, i see that now you always create the key `create_api_key` and do not get it